integration/test_alertrules.py

```diff
@@ -0,0 +1,62 @@
+def test_rules_add_user(connection):
+    response = connection.alerts.rules.add_user(
+        "d52cbfe0-f9de-468e-afbe-3c91037322da", 912249223544144039
+    )
+    assert response.status_code == 200
+
+
+def test_rules_get_all(connection):
+    response_gen = connection.alerts.rules.get_all()
+    for response in response_gen:
+        assert response.status_code == 200
+        break
+
+
+def test_rules_get_by_observer_id(connection):
+    observer_id = "d52cbfe0-f9de-468e-afbe-3c91037322da"
+    response = connection.alerts.rules.get_by_observer_id(observer_id)
+    assert response.status_code == 200
+
+
+def test_rules_get_all_by_name(connection):
+    response_gen = connection.alerts.rules.get_all_by_name("Test Alerts using CLI")
+    for response in response_gen:
+        assert response.status_code == 200
+        break
+
+
+def test_rules_get_page(connection):
+    response = connection.alerts.rules.get_page()
+    assert response.status_code == 200
+
+
+def test_rules_remove_user(connection):
+    response = connection.alerts.rules.remove_user(
+        "d52cbfe0-f9de-468e-afbe-3c91037322da", 912249223544144039
```

Comment:
We should make the real IDs (not the legacy IDs, but IDs like this one right here) `str` because that is what the API returns and so it is more expected.

Plus I think from a memory standpoint, the long integer have predefined set of bits (64?) and strings are just a 1 byte per character, not like memory would ever be an issue, but it seems like a better use of the data types anyway

Reply:
Not really a big deal though! Just more expected

Reply:
changed.